assets/src/scripts/store/seriesReducer.spec.js

```diff
@@ -0,0 +1,195 @@
+                    data: {
+                        stateToMerge: {},
+                        timeSeries: {
+                            ts1: {
+                                variable: 'varId2',
+                                points: [1, 2]
+                            }
+                        },
+                        variables: {
+                            'varId2': {
+                                oid: 'varId2',
+                                variableCode: {
+                                    value: 2
+                                }
+                            }
+                        }
+                    }
+                }
+            )).toEqual({
+                stateToMerge: {},
+                timeSeries: {
+                    ts: {
+                        variable: 'varId',
+                        points: [1]
+                    },
+                    ts1: {
+                        variable: 'varId2',
+                        points: [1, 2]
+                    }
+
+                },
+                variables: {
+                    'varId': {
+                        oid: 'varId',
+                        variableCode: {
+                            value: 1
+                        }
+                    },
+                    'varId2': {
+                        oid: 'varId2',
+                        variableCode: {
+                            value: 2
+                        }
+                    }
+                }
+            });
+        });
+    });
+
+    describe('RESET_TIMESERIES', () => {
+        it('If series is already empty will return an object with a requests object the key set to the empty object and an empty timeSeries property', () => {
+            expect(seriesReducer({}, {
+                type: 'RESET_TIMESERIES',
+                key: 'previous'
+            })).toEqual({
+                timeSeries: {},
+                requests: {
+                    previous: {}
+                }
+            });
+        });
+
+        it('If series contains a timeseries the series with the key is reset', () => {
+            const result = seriesReducer({
+                timeSeries: {
+                    ts: {
+                        variable: 'varId',
+                        tsKey: 'current',
+                        points: [1]
+                    },
+                    ts1: {
+                        variable: 'varId2',
+                        tsKey: 'compare',
+                        points: [1, 2]
+                    }
+                },
+                requests: {
+                    current: {
+                        queryInfo: 'current'
+                    },
+                    compare: {
+                        queryInfo: 'compare'
+                    }
+                }
+            }, {
+                type: 'RESET_TIMESERIES',
+                key: 'compare'
+            });
+            console.log(result);
```

Comment:
Should this console.log be here?

Reply:
Nope. I will remove it in the next PR. I'm going to go ahead and merge this one.